app/src/main/java/me/phh/treble/app/Style.kt

```diff
@@ -46,6 +46,29 @@ object Style: EntryStartup {
                     OverlayPicker.setOverlayEnabled(value, true)
                 }
             }
+            StyleSettings.iconPack -> {
+                val value = sp.getString(key, "")
+                val genericValue = value.toString().substringBeforeLast(".")
```

Comment:
Uh I hoped for something more generic.
Other icon_packs might want to override more packages? (like say override play store icon

Do you think this can be done?

Reply:
Hmm...
I don't think it can be done, not at least with overlays...
By "icon pack", it means something like [this](https://www.reddit.com/r/android_beta/comments/cm887w/i_found_builtin_system_icon_theme_in_q_beta_5/)
Maybe the launcher overlays are able to do this, I don't know

Reply:
I don't think it can be made, I already tried previously and it was just applying com.android.theme.icon_pack.name.android